pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -223,33 +221,19 @@ func Test_InFlightPods(t *testing.T) {
 			},
 			wantInFlightPods:   nil,
 			wantInFlightEvents: nil,
-		},
-		{
-			name:        "when SchedulingQueueHint is disabled, which queue to enqueue Pod should be decided without SchedulingQueueHint",
```

Comment:
Just moved to L391 with renaming to organize the order of test cases - tests of how we determine which queue to requeue are coming after tests of how we register inFlightPods/inFlightEvents.